### PR TITLE
fix(storefront): BCTHEME-214 Wishlist first option has no bottom outline when in focus

### PR DESCRIPTION
**Pull request by yurytut1993**
#### What?

In this pr focus state of wishlist items was updated. First option now has correct outline at the bottom. Also wishlist main link has correct outline when its focused and option items are opened 

#### Tickets / Documentation

[Ticket](https://jira.bigcommerce.com/browse/BCTHEME-214)

#### Screenshots (if appropriate)

wishlist_option_in_focus

### CODE REVIEW

**Review by bc-alexsaiannyi — APPROVED** [review]

File: assets/scss/components/foundation/dropdown/_dropdown.scss [hunk 1]
```diff
@@ -10,6 +10,8 @@
 }
 
 .dropdown-menu {
+    z-index: 1;
```
Comment (BC-tymurbiedukhin) [inline assets/scss/components/foundation/dropdown/_dropdown.scss:13]: Could we use zIndex tool map from _z-index.scss file or do we need exactly 1 and 2 values for the fix?

Comment (yurytut1993) [reply assets/scss/components/foundation/dropdown/_dropdown.scss:13]: We can use $zIndex-low and $zIndex-high from zIndex tool but we do not need so big values. Should we add more variables to the tool? z-index value of .dropdown-menu-button must be higher then .dropdown-menu

Comment (BC-tymurbiedukhin) [reply assets/scss/components/foundation/dropdown/_dropdown.scss:13]: I believe, if we can use current variables and this won't affect any new issue, we should do it)

Comment (yurytut1993) [reply assets/scss/components/foundation/dropdown/_dropdown.scss]: done

**Review by BC-tymurbiedukhin — APPROVED** [review]

**Pull request merged**